docs/strategy.md

```diff
@@ -0,0 +1,160 @@
+# Gas Cost Estimator approach strategy
+
+Here we organize thoughts on different approaches to consider.
+We focus on EVM for simplicity, but this should be substitutable by eWASM throughout the document.
```

Comment:
Ethereum call it just WASM

Reply:
i've seen ewasm is quite common, let's be specific. https://github.com/ewasm but I'll correct the spelling :)